OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -18,6 +18,8 @@ namespace OpenRA.Network
 {
 	static class UnitOrders
 	{
+		private const string ServerChatName = "Battlefield Control";
```

Comment:
Keep it here and nontranslated please.  This is not the time or way to implement translations. Removing the explicit private is sufficient.

Const variables should indeed start with a capital letter.